eng/targets/CSharp.Common.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>9.0</LangVersion>
```

Comment:
FYI @dotnet/aspnet-build. 9.0 should be pretty stable to use everywhere now.

Reply:
See you removed most `$(LangVersion)` settings elsewhere in the repo. Are any left❔